entity_client.ts

```diff
@@ -50,3 +68,113 @@ export function kind_crud_api(papiea_url: string, provider: string, kind: string
     }
     return crudder
 }
+
+// class based crud
+interface EntityObjectCRUD {
+    update(spec: Spec):Promise<EntityObjectCRUD>
+    delete():Promise<void>
+    refresh():Promise<EntityObjectCRUD>
+    invoke(procedure_name:string, input:any):Promise<any>
+}
+
+export class MutableEntityObject implements EntityObjectCRUD{
```

Comment:
Mutable object is not adding much value (as you had mentioned). Plus it adds race conditions in multi-threaded programs. So, let's skip.